tests/snippets/builtin_format_map.py

```diff
@@ -0,0 +1,64 @@
+from testutils import AssertRaises
+
+
+def test_format_map():
+    # The following tests were performed in Python 3.7.5:
```

Comment:
Could you move these tests to `tests/snippets/strings.py`? And you don't need the separate test function; our tests right now are a lot less structured than most Python tests 😁